### PR TITLE
feat(ProxyConnectionService): use JitsiConnection to fetch TURN credentials

### PR DESCRIPTION
**Pull request by paweldomas**

### CODE REVIEW

File: modules/proxyconnection/ProxyConnectionService.js [hunk 1]
```diff
@@ -34,12 +37,21 @@ export default class ProxyConnectionService {
...
          * Holds a reference to the collection of all callbacks.
          *
          * @type {Object}
          */
-        this._options = options;
+        this._options = {
+            iceConfig: jitsiConnection
+                && jitsiConnection.xmpp.connection.jingle.p2pIceConfig,
+            ...otherOptions
+        };
```
Comment (virtuacoplenny) [inline modules/proxyconnection/ProxyConnectionService.js]: We'll handle refreshing some other day, yeah?

Comment (virtuacoplenny) [reply modules/proxyconnection/ProxyConnectionService.js]: Also, are we guaranteed to have p2pIceConfig defined?

Comment (paweldomas) [reply modules/proxyconnection/ProxyConnectionService.js]: Yes, I was thinking to do that in the xmpp module. In spot they will be fetched each time new screensharing connection is created ?

Comment (paweldomas) [reply modules/proxyconnection/ProxyConnectionService.js]: we don't guarantee, but then it should give `iceConfig` `undefined` and then it should default to `DEFAULT_STUN_SERVERS` as far as I understand

**Pull request merged**